### DISCUSSION TITLE
Tortoise SVN client no longer working?

### DISCUSSION THREAD

**Discussion by videogameboy76** [community/community#108105] [Programming Help] [answered]
### Body

I have a number of repositories, and a couple of them I develop on Windows so I use Tortoise SVN for those.  It is no longer working, though.

The others are on Ubuntu, so I use the Git CLI there with no issues.

When I try to access the repository using Tortoise SVN, it says "Unable to connect to a repository at URL 'https://github.com/videogameboy76/project.git' path not found.

This affects both my public and private repositories.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by videogameboy76** [comment]
Some additional info: I was using TortoiseSVN 1.9.7, but even after updating to 1.14.6 it didn't change anything.

I was using a personal token (forced after password use was eliminated). but I don't even get a prompt for authentication after removing my auto-login - the location simply doesn't appear to exist per the SVN client.  I disabled my firewall to see if that was the culprit, and it was not.

I tried using GitHub Desktop, and it works.  I will continue to use GitHub Desktop until such time that the SVN access works again, as I find it the simplest interface by far.

**Comment by Microcom-Bjarne** [comment]
You could try TortoiseGIT. I use both since I've got access to quite a lot of older svn repos. And I have the same error you are having. At least I think it's the same - or close related. 
In many of my SVN repos I've got svn:external declared that links from svn to git repositories. But since some days I've been unable to checkout these repos. The error I get is - 
Error: Unable to connect to a repository at URL  
Error:  'https://github.com/<my personal git>/<Repo>.git'  
....

Hope anyone has some more information on this issue.

**Comment by WolfeStrider** [comment, marked as answer]
Subversion has been unceremoniously sunset :/
https://github.blog/changelog/2024-01-08-subversion-has-been-sunset/

**Reply by videogameboy76** [reply]
Thank you for the reply.  Good to know it wasn't anything I neglected on my end.  Looks like Github Desktop will be my new go-to client for Windows.  It adds a step, but it is actually rather nice to see a preview of the changes in the actual file, including supported binary file types (graphics, etc.).

**Comment by Mindflayer86** [comment]
I am quite astonished that this did not spark a far bigger outcry.
Stumbled upon the same problem as op and it was driving me nuts. Tried different TortoiseSVN versions. Tried different SVN versions.
This breaks my entire workflow! It was so nice to keep my professional SVN projects and my github source code collections, comfortably in sync and usable with the same Source Control Client. Now its gone.
I hope they have a really, really, really pressing reason to drop SVN support and its not some bullshit management decision.
Github got a lot worse this year...

**Comment by schellingb** [comment]
I spent the first few months in 2024 to make a replacement GUI client for my workflow, which used to depend on the fantastic TortoiseSVN. I now released that client for myself and anyone who is interested in a lean, simplified git GUI experience for Windows in a single EXE file under 1 MB with no dependencies (no bloated "git for windows" needed), and I call it: CozyGit, available here: https://github.com/schellingb/CozyGit
